common/model-views.cpp

```diff
@@ -2810,7 +2851,8 @@ namespace rs2
             auto index = 0;
             while (ppf.resulting_queue.poll_for_frame(&f) && ++index < ppf.resulting_queue_max_size)
             {
-                last_frames[f.get_profile().unique_id()] = std::move(f);
+                auto id = f.get_profile().unique_id();
```

Comment:
Alternatively, just remove the std::move, it's an unnecessary optimization

Reply:
Agree